node/hyperbahn/service_proxy.js

```diff
@@ -296,7 +296,7 @@ function refreshServicePeer(serviceName, hostPort) {
     var self = this;
 
     var peer = self.getServicePeer(serviceName, hostPort);
-    peer.connect();
+    peer.connect(true);
```

Comment:
This makes sense.  Eventually I'd like to see us also shutdown any incoming connections once the new outgoing connection is identified, so maybe something like:
```
var conn = peer.connect(true);
if (peer has any incoming connections) {
  conn.identifiedEvent.on(thenCloseAnyIncomingConnections);
}
```